zipkin-ui/static/nav.properties

```diff
@@ -1,4 +1,5 @@
 nav.inves = Investigate system behavior
 nav.find = Find a trace
 nav.dep = Dependencies
-nav.search = Go to trace
\ No newline at end of file
+nav.search = Go to trace
+nav.viewSaved = View Saved Trace
```

Comment:
@MrGlaucus @gianarb @drolando can you contribute chinese and italian text for this?

Reply:
I will work on the translation as the merge is done.